### PR TITLE
[ITSADSSD-53175] New hero component

### PR DESCRIPTION
**Pull request by jptaranto**
This is based off https://github.com/uq-its-ss/design-system/pull/452 as it relies on the new layout CSS.

The hero component has been refactored. For now this just includes the homepage and level 1/2 variations. The basic hero will be split out into a new `.uq-hero-basic` component, and included with the `@uqds/hero` package.

Screenshot 2024-01-19 at 11 53 25 am

Screenshot 2024-01-19 at 12 13 01 pm

Screenshot 2024-01-19 at 11 54 17 am

### CODE REVIEW

**Comment by jptaranto** [comment]
Screenshot 2024-01-23 at 9 25 45 am

File: packages/storybook-html/stories/components/hero/hero.docs.mdx [hunk 1]
```diff
@@ -8,84 +8,72 @@ import StatusBadge from "../../../.storybook/custom/components/StatusBadge";
...
-    </div>
+    <img src="/images/hero/hero-example-1.jpg" alt="" role="presentation" />
   </div>
-  <div class="uq-hero__content">
-    <div class="uq-hero__text">
-      <div class="uq-hero__title">Start postgraduate study online</div>
-      <p class="uq-hero__subtext">Apply by June 30</p>
-      <p><a class="uq-button" href="#">Learn more</a></p>
+  <div class="uq-container">
+    <div class="uq-hero__content">
```
Comment (uqnm) [inline packages/storybook-html/stories/components/hero/hero.docs.mdx:39]: would these be better without the wrapping elements eg <p class="uq-hero__description">Description</p> and remove uq-hero__button?

**Review by uqnm — APPROVED** [review]
this looks good overall. other than my comment my minor notes (more for me) is that depending on the order of things we need to make updates to color variables based on https://github.com/uq-its-ss/design-system/pull/445

File: packages/storybook-html/stories/components/hero/hero.docs.mdx (hunk 1 above)
Comment (jptaranto) [reply packages/storybook-html/stories/components/hero/hero.docs.mdx:39]: The wrappers provide surety around spacing without having to rely on the child elements - so it hopefully makes things a little more robust.

**Review by Bcrunk — APPROVED** [review]

**Pull request merged**